### PR TITLE
added a link to ex-mode in readme

### PR DESCRIPTION
**Pull request by flyingfisch**
I added a link to the ex-mode plugin in the readme, since the two projects are closely related.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +25,8 @@ more often than other developers. Adding a feature can be quick and easy. Check
 out the [closed pull requests](https://github.com/atom/vim-mode/pulls?direction=desc&page=1&sort=created&state=closed)
 to see examples of community contributions. We're looking forward to yours, too.
 
+
+
```
Comment (maxbrunsfeld) [inline README.md]: Could you :fire: these extra line breaks?

**Comment by maxbrunsfeld** [comment]
Great idea. Looks good except for the minor feedback above.

**Comment by flyingfisch** [comment]
Line breaks are gone. ;)

**Comment by maxbrunsfeld** [comment]
Thanks!

**Pull request merged**